b2sdk/encryption/types.py

```diff
@@ -0,0 +1,31 @@
+######################################################################
+#
+# File: b2sdk/encryption/types.py
+#
+# Copyright 2021 Backblaze Inc. All Rights Reserved.
+#
+# License https://www.backblaze.com/using_b2_code.html
+#
+######################################################################
+
+from enum import Enum, unique
+from typing import NewType
+
+EncryptionKey = NewType('EncryptionKey', bytes)
+
+
+@unique
+class EncryptionAlgorithm(Enum):
+    AES256 = 'AES256'
+
+
+@unique
+class EncryptionMode(Enum):
+    UNKNOWN = None  #: unknown encryption mode (sdk doesn't know or used key has no rights to know)
+    NONE = "none"  #: no encryption (plaintext)
+    SSE_B2 = 'SSE-B2'  #: server-side encryption with key maintained by B2
+    SSE_C = 'SSE-C'  #: server-side encryption with key provided by the client
```

Comment:
If it's not implemented we should remove/comment it

Reply:
We are keeping it so that `ls` like operations can see files encrypted with SSE-C (by other tools or by future versions of b2sdk)